clustered/server/ehcache-service/src/test/java/org/ehcache/clustered/server/store/ServerStoreTest.java

```diff
@@ -78,6 +79,15 @@ public void testGetMappingExists() throws Exception {
     assertThat(chain, hasPayloads(1L));
   }
 
+  @Test
```

Comment:
You can move this test to the offheap specific subclass where you know the class under test is a server side store, and not a proxy.